### ISSUE TITLE
using legacy keys. Meepo *MegaMeepo* Aghanism

### ISSUE THREAD

**Issue by xAgarwood** [ValveSoftware/Dota2-Gameplay#8179]
### Description

Helllo im using legacy keys on dota 2 there no hotkey in skill MegaMeepo Aghanism scepter. i hope u put hotkey MegaMeepo.

### Screenshots

![123123123](https://user-images.githubusercontent.com/131469499/233667719-3a801aa2-7372-41fe-bdac-9630fea09b11.jpg)

**Comment by Neacroe** [comment]
Yeah, mega meepo and mega meepo throw cannot be bound to diff keys. It always makes them the same despite showing otherwise.

**Comment by dnzguir** [comment]
up to this, this is needed to be fixed.

**Comment by Avarise831** [comment]
#8082
